mcu/stm32f103.S

```diff
@@ -105,6 +105,75 @@ __isr_vector:
 /* Peripheral interrupts are not used */
     .size __isr_vector, . - __isr_vector
 
+#if (DFU_VERIFY_CHECKSUM != _DISABLE)
+    .section    .text
```

Comment:
Common .text section. This table will takes a memory for both _SMALL and _FAST. Even with --gc-sections linker option.